service/penalties.go

```diff
@@ -121,7 +125,7 @@ func generateTransactionListFromE5Response(e5Response *e5.GetTransactionsRespons
 			return nil, err
 		}
 		listItem.IsPaid = e5Transaction.IsPaid
-		listItem.Kind = "late-filing-penalty#late-filing-penalty"
+		listItem.Kind = penaltyDetailsMap.Details[penaltyType]["ResourceKind"]
```

Comment:
As we're only using the ResourceKind field, would it be preferable to work this out upstream and pass in that field only, instead of always passing in the full penaltyDetailsMap?

Or do we expect other fields within this map to be used here in future iterations of this?

Reply:
I honestly don't know we will need more :) I see what you mean if it is only the one value. I am happy to keep this as is for now, for ease of following through the logic i.e. just following the penatlyDetailsMap parameter through, but I think it is something to revisit.